tools/integration-test/src/tests/fee/filter_fees.rs

```diff
@@ -0,0 +1,408 @@
+use std::collections::HashMap;
+
+use ibc_relayer::config::filter::{ChannelPolicy, FeeInformation, FeesFilters, FilterPattern};
+use ibc_relayer::config::PacketFilter;
+use ibc_relayer_types::core::ics04_channel::version::Version;
+use ibc_test_framework::prelude::*;
+use ibc_test_framework::util::random::random_u128_range;
+
+#[test]
+fn test_filter_incentivized_fees_relayer() -> Result<(), Error> {
+    run_binary_channel_test(&FilterIncentivizedFeesRelayerTest)
+}
+
+#[test]
+fn test_filter_by_channel_incentivized_fees_relayer() -> Result<(), Error> {
+    run_binary_channel_test(&FilterByChannelIncentivizedFeesRelayerTest)
+}
+
+/*#[test]
```

Comment:
Why is this commented out?

Reply:
This is the test in case we allow filtering of acknowledgements. As the helper function to filter them, `_get_incentivized_for_write_acknowledgement `, was kept I didn't remove the test either. But I can remove it